jest.config.js

```diff
@@ -10,4 +10,7 @@ module.exports = {
     '\\.(svg|png)$': '<rootDir>/empty-module.js',
   },
   snapshotSerializers: ['preact-render-spy/snapshot'],
+  // The webpack config folder for webpacker is excluded as it has a test.js file that gets
+  // picked up by jest if this folder is not excluded causing a false negative of a test suite failing.
+  testPathIgnorePatterns: ['/node_modules/', './config/webpack'],
```

Comment:
The `node_modules` folder is included as well as that's the default value. See [testPathIgnorePatterns](https://jestjs.io/docs/en/configuration#testpathignorepatterns-arraystring)